ContosoHelpdeskChatBot/ContosoHelpdeskChatBot/Dialogs/MainDialog.cs

```diff
@@ -12,17 +12,19 @@ namespace ContosoHelpdeskChatBot.Dialogs
 {
     public class MainDialog : WaterfallDialog
     {
-        private const string InstallAppOption = "Install Application (install)";
```

Comment:
this is by design to be text so that users can initiate conversation over SMS as well as email